### ISSUE TITLE
Menu of links to other pages

### ISSUE THREAD

**Issue by pwellner** [openiaq/map-website#4]
Much like existing site's menu button, leading to a variety of other pages. 
Perhaps this should be easy to customize without having to rebuild.

**Comment by pwellner** [comment]
@AurelWu  Perhaps the map could be published as a React component so that the indoorCo2map site can provide its own separate navigation bar above the map component?

**Comment by AurelWu** [comment]
yep totally makes sense to have that separate. sorry for the late reply.
